### PR TITLE
Add gen2 auth manual configure

### PR DESCRIPTION
**Pull request by josefaidt**
#### Description of changes:

follow-up to #7613 

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by ykethan — DISMISSED** [review]

**Review by reesscot — DISMISSED** [review]

File: src/directory/directory.mjs [hunk 1]
```diff
@@ -162,6 +162,9 @@ export const directory = {
                 {
                   path: 'src/pages/[platform]/build-a-backend/auth/grant-access-to-auth-resources/index.mdx'
                 },
+                {
+                  path: 'src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx'
```
Comment (renebrandel) [inline src/directory/directory.mjs:166]: Maybe a more verbose subpath "use-existing-cognito-user-pool"

File: src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+Amplify Auth can be configured to use a existing resources. If you are in a team setting or part of a company that has previously created auth resources, you have a few different options to configure your application to use existing auth resources:
+
+<InlineFilter filters={["angular", "javascript", "nextjs", "react", "react-native", "vue"]}>
+
+- [Add auth configuration as an output of the backend](#add-auth-as-a-backend-output)
+- [Configure the client library directly (without an Amplify backend)](#configure-the-client-library-directly)
+
+</InlineFilter>
+{/* merge content after hosting outputs schema */}
```
Comment (renebrandel) [inline src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx:40]: What does this mean? Doesn't the console already provide the outputs?

File: src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+```ts title="amplify/backend.ts"
+import { defineBackend } from "@aws-amplify/backend"
+import { UserPool, UserPoolClient } from "aws-cdk-lib/aws-cognito"
+
+/**
+ * @see https://docs.amplify.aws/react/build-a-backend/ to add storage, functions, and more
+ */
+const backend = defineBackend({})
+
+const authStack = backend.createStack("ExistingAuth")
```
Comment (renebrandel) [inline src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx:103]: Does this approach line up with the Storage approach? What about customers that don't want a "dummy stack" to add these outputs?

**Review by renebrandel — CHANGES_REQUESTED** [review]

File: src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx (hunk 2 above)
Comment (josefaidt) [reply src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx:40]: we are unable to configure mobile client libraries directly, and need to document how to create this outputs file manually if you do not have an amplify backend. Ideally that can be authored by using the JSON schema

File: src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx (hunk 3 above)
Comment (josefaidt) [reply src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx:103]: that is in the previous example

File: src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx [hunk 4]
```diff
@@ -0,0 +1,206 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Use existing resources',
+  description: 'Learn how to use existing auth resources',
```
Comment (renebrandel) [inline src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx:5]: update this to Cognito as well

File: src/directory/directory.mjs (hunk 1 above)
Comment (josefaidt) [reply src/directory/directory.mjs:166]: updated

File: src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx (hunk 4 above)
Comment (josefaidt) [reply src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx:5]: updated

**Review by aspittel — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/auth/use-existing-cognito-resources/index.mdx [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+
+{/* pending hosted outputs schema */}
+```json title="amplify_outputs.json"
+{
+  "$schema": "..."
+}
+```
+
+</InlineFilter>
+</InlineFilter>
```
Comment (reesscot) [inline src/pages/[platform]/build-a-backend/auth/use-existing-cognito-resources/index.mdx:202]: extra ending inlinefilter?
```suggestion
```

Comment (josefaidt) [reply src/pages/[platform]/build-a-backend/auth/use-existing-cognito-resources/index.mdx:203]: added a comment to it

**Review by ykethan — APPROVED** [review]

**Review by hdworld11 — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/auth/use-existing-cognito-resources/index.mdx [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+      meta
+    }
+  };
+}
+
+Amplify Auth can be configured to use a existing resources. If you are in a team setting or part of a company that has previously created auth resources, you have a few different options to configure your application to use existing auth resources:
+
+<InlineFilter filters={["angular", "javascript", "nextjs", "react", "react-native", "vue"]}>
+
+- [Add auth configuration as an output of the backend](#add-auth-as-a-backend-output)
```
Comment (swaminator) [inline src/pages/[platform]/build-a-backend/auth/use-existing-cognito-resources/index.mdx:36]: How should I decide which option to pick? Its unclear.

File: src/pages/[platform]/build-a-backend/auth/use-existing-cognito-resources/index.mdx [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+Amplify Auth can be configured to use a existing resources. If you are in a team setting or part of a company that has previously created auth resources, you have a few different options to configure your application to use existing auth resources.
```
Comment (renebrandel) [inline src/pages/[platform]/build-a-backend/auth/use-existing-cognito-resources/index.mdx:32]: grammar.
```suggestion
Amplify Auth can be configured to use an existing Amazon Cognito user pool and identity pool. If you are in a team setting or part of a company that has previously created auth resources, you have a few different options to configure your application to use existing auth resources.
```

**Review by aspittel — APPROVED** [review]

**Review by ykethan — APPROVED** [review]

**Pull request merged**